global.json

```
@@ -15,7 +15,7 @@
     "Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk": "5.0.0-beta.20280.1",
     "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20280.1",
     "Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk": "5.0.0-beta.20280.1",
-    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20280.1",
+    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20304.1",
```

Can we try this version?

```suggestion
    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20308.1",
```